### PR TITLE
Add 1.25 to release schedule for 1.25 release

### PR DESCRIPTION
**Pull request by reylejano**
This PR adds 1.25 to the releases schedule for the 1.25.0 release
/milestone 1.25
/cc @kcmartin @cici37

### CODE REVIEW

**Comment by sftim** [comment]
LGTM

File: data/releases/schedule.yaml
Comment (sftim) [inline data/releases/schedule.yaml:7]: These aren't parsing as dates.

Comment (cici37) [reply data/releases/schedule.yaml:7]: I guess it is ok to leave it empty as of now. 
And I assume we don't need cherryPickDeadline?

Comment (reylejano) [reply data/releases/schedule.yaml]: initially I left out `next`, `cherryPickDeadline`, `targetDate`, and `endOfLifeDate` and the build failed
Build also failed with a string ("TBD") instead of valid dates

Comment (reylejano) [reply data/releases/schedule.yaml]: Build succeeded with
```
- release: 1.25
  releaseDate: 2022-08-23
  next: 1.25.1
  cherryPickDeadline: 2022-09-09
  targetDate: 2022-09-14
  endOfLifeDate: 2023-10-31
```

**Comment by reylejano** [comment]
Preview:
https://deploy-preview-36174--kubernetes-io-vnext-staging.netlify.app/releases/

File: data/releases/schedule.yaml
Comment (reylejano) [reply data/releases/schedule.yaml]: Looks like we have to have entries for  `next`, `cherryPickDeadline`, `targetDate`, and `endOfLifeDate` with valid dates

**Comment by tengqm** [comment]
Looks like we need to manually add a subsection `1.25` to `k8s.io/releases/patch-releases/`, or else the link target on https://deploy-preview-36174--kubernetes-io-vnext-staging.netlify.app/releases/ has 404.

**Comment by reylejano** [comment]
> Looks like we need to manually add a subsection `1.25` to `k8s.io/releases/patch-releases/`, or else the link target on https://deploy-preview-36174--kubernetes-io-vnext-staging.netlify.app/releases/ has 404.

added a subsection for 1.25 to /releases/patch-releases.md

**Comment by sftim** [comment]
I'm happy to help bypass the Tide merge block once this has LGTM / approvals from SIG Release folks.

**Comment by sftim** [comment]
Not required for _this_ PR, but we should soon also drop https://k8s.io/releases/patch-releases/#1-22 I think (move it into https://kubernetes.io/releases/patch-releases/#non-active-branch-history)

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Comment by palnabarun** [comment]
As a follow-up, I will make the patch release page use the datafile instead of manually updating the page everytime.

**Pull request merged**
